package.json

```diff
@@ -98,8 +98,7 @@
     "shelljs-nodecli": "~0.1.0",
     "sinon": "^1.17.2",
     "temp": "^0.8.3",
```

Comment:
If temp doesn't cleanup after itself, do we need it? I would rather remove it, if it's not necessary.